tests/test_passes.py

```diff
@@ -733,6 +732,68 @@ def test_nested_elimination(self):
         pyrtl.working_block().sanity_check()
 
 
+class TestSynthPasses(NetWireNumTestCases):
+    in1: pyrtl.Input
+    in2: pyrtl.Input
+    out: pyrtl.Output
+
+    def setUp(self):
+        pyrtl.reset_working_block()
+        self.in1 = pyrtl.Input(bitwidth=5, name='in1')
+        self.in2 = pyrtl.Input(bitwidth=5, name='in2')
+        self.out = pyrtl.Output(bitwidth=5, name='out')
+
+    def run_nand_synth(self, values: dict[tuple[int, int], int]):
```

Comment:
Add a comment that explains the structure of `values` (it maps from input values for `(in1, in2)` to expected value of `out`)